lib/src/modules/compiler/template_parser/lib/src/visitor.dart

```diff
@@ -0,0 +1,29 @@
+library angular2_template_parser.src.visitor;
+
+import 'ast.dart';
+
+part 'visitor/unparser.dart';
+
+/// [Visitor] interface is provided to an [NgAstNode] node via
```

Comment:
Try to always have a single standalone line in a Dartdoc.

/// Allows visiting an [NgAstNode].
///
/// ...